config/jobs/kubernetes-sigs/cluster-api-operator/cluster-api-periodics-main.yaml

```diff
@@ -0,0 +1,24 @@
+periodics:
+- name: periodic-cluster-api-operator-test-main
+  interval: 1h
+  decorate: true
+  labels:
+    preset-service-account: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api-operator
+    base_ref: main
+    path_alias: sigs.k8s.io/cluster-api-operator
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211217-ea95cec1d4-1.23
+      command:
+      - "./scripts/ci-test.sh"
+      resources:
+        requests:
+          cpu: 7300m
```

Comment:
Where did this value come from? Seems quite high

Reply:
I took it from core capi job config

Reply:
We can probably use the default unless we know we need this much for running specific tests, might be worth doing a git blame on the capi config and seeing if there's any reasoning for this specific value

Reply:
https://github.com/kubernetes/test-infra/pull/18439, we can drop the request right now and see if the default works